### DISCUSSION TITLE
How to Ingnore Keyboard Shotcut CTRL+P

### DISCUSSION THREAD

**Discussion by jayeshsolankiii** [stephanrauh/ngx-extended-pdf-viewer#2404] [Q&A] [answered]
How to Ignore Keyboard Shotcut CTRL+P

**Comment by stephanrauh** [comment, marked as answer]
I've investigated the topic a few years ago, and the sad truth is you can't suppress `CTRL` + `P`. Try as I might, the browser would always ignore my attempt to override this key combination.

I've documented this (albeit slightly hidden) on https://pdfviewer.net/extended-pdf-viewer/keyboard.

**Reply by jayeshsolankiii** [reply]
Thanks @stephanrauh
As we discussed https://github.com/stephanrauh/ngx-extended-pdf-viewer/issues/2403
